server/config/persist_options.go

```diff
@@ -200,6 +200,12 @@ const (
 	hotRegionScheduleLimitKey      = "schedule.hot-region-schedule-limit"
 	schedulerMaxWaitingOperatorKey = "schedule.scheduler-max-waiting-operator"
 	enableLocationReplacement      = "schedule.enable-location-replacement"
+	// it's related to schedule, but it's not an explicit config
+	enableTiKVSplitRegion = "schedule.enable-tikv-split-region"
```

Comment:
Do we need to add it to ScheduleConfig?

Reply:
not necessary for now, it's not a real config(it's always true except during `br` operation)

can be deprecated when pd support one-api to pause all schedulers.

Reply:
I think it's ok to `ScheduleConfig`, in some scenarios, we also need to disable split. such as recovery

Reply:
https://github.com/tikv/pd/pull/5485/commits/198c7cc47bbfc28d22cb1d5fcc0ba243a7f887a4 , it's visible internally for now(so cannot disable it without ttl)